test_libs/pyspec/requirements-testing.txt

```diff
@@ -2,3 +2,4 @@
 pytest>=3.6,<3.7
 ../config_helpers
 flake8==3.7.7
+pytest-cov
```

Comment:
yay for coverage reports. Don't forget to `make install_test` to enable it. Good for changelog.